docs/UserGuide.md

```diff
@@ -187,18 +188,18 @@ Examples:
 
 ### Deleting a customer : `deletecustomer`
 
-Deletes the specified customer from the address book.
+Deletes the specified customer contact and order list from the address book.
 
 Command aliases: `deletec`<br>
 Format: `deletecustomer CUSTOMER_INDEX`
 Examples:
-* `listcustomer` followed by `deletecustomer 2` deletes the 2nd customer in the address book.
-* `findcustomer Betsy` followed by `deletec 1` deletes the 1st customer in the results of the `find` command.
+* `listcustomer` followed by `deletecustomer 2` deletes the 2nd customer contact and order list in the address book.
+* `findcustomer Betsy` followed by `deletec 1` deletes the 1st customer contact and order list in the results of the `find` command.
 
 <box type="important">
 
 ##### Constraints
-* Deletes the customer at the specified `CUSTOMER_INDEX`.
+* Deletes the customer and the customer's order list at the specified `CUSTOMER_INDEX`.
```

Comment:
his order list?

Reply:
```suggestion
* Deletes the customer at the specified `CUSTOMER_INDEX`.
```
Since this line specifically refers to the target of the `CUSTOMER_INDEX`, not mentioning the order list should be fine.